frontend/src/stores/search.ts

```diff
@@ -508,7 +485,7 @@ export const useSearchStore = defineStore("search", {
       filterCategory: FilterCategory
     ): boolean | undefined {
       if (!["licenseTypes", "licenses"].includes(filterCategory)) {
-        return
+        return false
```

Comment:
Does this change allow us to remove `undefined` from the return type of this function on line 486?